### ISSUE TITLE
pepper 0.7.0 -> AttributeError: 'module' object has no attribute 'JSONDecodeError' with python2.7

### ISSUE THREAD

**Issue by robin-wittler** [saltstack/pepper#154]
Python2.7 does not have json.decoder.JSONDecodeError

```Python
Traceback (most recent call last):
  File "/var/lib/jenkins/.local/bin/pepper", line 53, in __call__
    for exit_code, result in self.cli.run():
  File "/var/lib/jenkins/.local/lib/python2.7/site-packages/pepper/cli.py", line 600, in run
    load = self.parse_cmd()
  File "/var/lib/jenkins/.local/lib/python2.7/site-packages/pepper/cli.py", line 473, in parse_cmd
    except json.decoder.JSONDecodeError:
AttributeError: 'module' object has no attribute 'JSONDecodeError'
```

**Comment by robin-wittler** [comment]
see https://github.com/saltstack/pepper/pull/155

**Comment by gtmanfred** [comment]
Fix released in 0.7.1

Thanks!
Daniel
